src/cleanup_rules/swift/edges.toml

```diff
@@ -18,6 +18,24 @@ scope = "Function"
 from = "delete_variable_declaration"
 to = ["replace_identifier_with_value", "delete_parent_assignment"]
 
+[[edges]]
+scope = "Class"
+from = "delete_field_initialisation_init"
+to = [
+  "replace_self_identifier_with_value",
+  "replace_identifier_with_value",
+  "delete_field_declaration",
```

Comment:
why not delete the assignments too?

Reply:
Also can we abstract these rules to groups ? And have edges just between the groups?